packages/graphql-hooks/package.json

```diff
@@ -38,6 +38,7 @@
     "rollup": "1.5.0",
     "rollup-plugin-babel": "4.3.2",
     "rollup-plugin-node-resolve": "4.0.1",
+    "rollup-plugin-replace": "2.1.0",
```

Comment:
Was this missing? I can add to #109 if so

Reply:
It was